__test__/server/api/campaign/campaign.test.js

```diff
@@ -1306,7 +1306,7 @@ describe("per-campaign phone numbers", async () => {
   });
 
   it("allocates numbers to a campaign and creates a messaging service on start", async () => {
-    const [phoneId] = await r.knex("owned_phone_number").insert(
+    const [res] = await r.knex("owned_phone_number").insert(
```

Comment:
does this need to be `res.id` or do we need the whole object here?

Reply:
`await r.knex(...).insert(...)` returns an object with a format of `[ { id: 1 } ]`

I think this is the cleanest way to pull `id` out of `res`. Do you have a different suggestion?